cli/lib/exec/run.js

```diff
@@ -45,6 +45,10 @@ const processRunOptions = (options = {}) => {
 
   const args = ['--run-project', options.project]
 
+  if (options.autoCancelAfterFailures || options.autoCancelAfterFailures === false) {
+    args.push('--auto-cancel-after-failures', options.autoCancelAfterFailures)
+  }
```

Comment:
I'd only expect this to be passed if autoCancelAfterFailures was not false

Reply:
It can be false, so the cloud can handle it and override the value set on the project settings

Reply:
👍🏻 I think I'd expect the server code to handle this by setting it to false without needing to pass `--auto-cancel-after-failures=false` from the CLI

Reply:
you can pass `--no-auto-cancel-after-failures` and it would set the value to `false`

Reply:
@emilyrohrbough no this is necessary... you could have globally **enabled** auto cancellation in the cloud, and you want to override that by turning it **off** in the CLI

Reply:
Gottchya. 👍🏻 The flag definition should be updated to reflect this; it's not clear is tied to globally Cloud configuration when record mode is enabled.